syntax = "proto3";

package syft.core.node.common.service;

import "proto/core/common/common_object.proto";
import "proto/core/io/address.proto";

message ObjectSearchPermissionUpdateMessage {
  syft.core.io.Address address = 1;
  bytes target_verify_key = 2;
  syft.core.common.UID target_object_id = 3;
  bool add_instead_of_remove = 4;
}
